apps/resources-wcm/src/main/webapp/javascript/eXo/wcm/frontoffice/public/ShareContent.js

```diff
@@ -123,6 +123,7 @@
       gj(".PopupContent .uiActionBorder .btn-primary").attr("disabled","disabled");
       gj("#shareActionBtn").trigger("click");
     });
+    location.reload(true);
```

Comment:
This is a workaround, a popup should be displayed indicating that the file was shared without loosing the File Explorer application in background. Please check the action in webUI and see why the file explorer disappears after sharing the document.